XouDevSpec/Classes/Helper/ObjCAnimeClass.h

```diff
@@ -0,0 +1,35 @@
+//
+//  ObjCAnimeClass.h
+//  XouTube
+//
+//  Created by Xander Schoeman on 2020/03/22.
+//  Copyright © 2020 Xander Schoeman. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ObjCAnimeClass : NSObject
+-(instancetype)initWith:(NSString *)mal_id :(NSString *)url :(NSString *)image_url :(NSString *)title :(NSString *)airing :(NSString *)synopsis :(NSString *)type :(NSString *)episodes :(NSString *)score :(NSString *)start_date :(NSString *)end_date :(NSString *)members :(NSString *)rated;
```

Comment:
This could be a struct

Reply:
Definitely, that's a massive init lol

Reply:
hehe, I actually tried to use it differently but it seems that I might just remove it and remake the whole objective c thing xD. But yeah you both make a good point it was also a pain typing it haha.